libs/accounts/email-sender/src/email-sender.ts

```diff
@@ -255,7 +263,27 @@ export class EmailSender {
         // Legacy auth-server implementation provided these, but they are not valid nodemailer options...
         // preview: email.preview,
         // xMailer: false,
-      });
+      };
+
+      let info: nodemailer.SentMessageInfo; // internally this is just `= any`
+      const retryIgnoreTemplates = this.config.retry?.ignoreTemplates || [];
+      const isIgnoredTemplate = retryIgnoreTemplates.includes(email.template);
+      const { maxRetries: maxAttempts, backoffMs } = this.config.retry || {
+        maxRetries: 0,
+        backoffMs: 0,
+      };
+      const canRetry = maxAttempts > 1 && backoffMs > 0 && !isIgnoredTemplate;
```

Comment:
I'd maybe add a guard rail to make sure maxRetries is a sane value, e.g. greater than zero and less than 10.